java/sage/miniclient/MiniClient.java

```diff
@@ -15,6 +15,15 @@
  */
 package sage.miniclient;
 
+import java.io.BufferedOutputStream;
```

Comment:
You can remove a bunch of these....File, PrintWriter, logging.*. They aren't referenced later.

Reply:
done